src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -29,8 +31,18 @@ public static partial class Certificates
 
             private static X509Certificate2Collection GetCertificateCollection(string certificateFileName)
             {
+                // On Windows, .Net Core applications should not import PFX files in parallel to avoid a known system-level race condition.
+                // This bug results in corrupting the X509Certificate2 certificate state.
+                if (!m.WaitOne(MutexTimeout))
+                {
+                    throw new InvalidOperationException("Cannot acquire the global certificate mutex.");
```

Comment:
It might be better to use Assert.True(false, "Cannot acquire the global certificate mutex.").  Throwing an exception when running the Xunit tests crashes them in a bad way compared to just failing a single test using the Assert.*